.changeset/chilled-timers-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nacelle/storefront-sdk-plugin-commerce-queries': minor
+---
+
+Adds a `content` method to fetch the `allContent` data. Unlike the `content` method in the v1 sdk, this returns a result object with the signature `{error, data}` similar to how the `query` method works in the v2 sdk to allow users more control over error handling.
```

Comment:
Recommend rephrasing to avoid confusion between `@nacelle/storefront-sdk@1.x` and "the other v1 SDK," `@nacelle/client-js-sdk`.

```suggestion
Adds a `content` method to fetch the `allContent` data. Unlike the `content` method in the `@nacelle/storefront-sdk@1.x`, this returns a result object with the signature `{ error, data }` similar to how the `query` method works in `@nacelle/storefront-sdk@2.x` to allow users more control over error handling.
```